src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -356,6 +359,48 @@ private void conjureSnapshotTablesFromRowIds(SnapshotRequestRowIdModel requestRo
         }
     }
 
+
+    private void conjureSnapshotTablesFromDatasetTables(Snapshot snapshot,
```

Comment:
Are you making a semantic difference between `conjure` and `create`?

Reply:
The semantic difference already exists in other snapshot workflows. This may be part of my later cleanup ticket

Reply:
I was being whimsical when I named it...

Reply:
I was being whimsical when I named it...